Helper.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+public class Helper
+{
+    protected List<char> _listOfLettersToCount;
+    protected List<char> _listOfNumberToCount;
+    private List<char> _illegalalCharacters;
+    private int? _result;
+
+    public Helper()
+    {
+        _listOfLettersToCount = ['a', 'b', 'c'];
+        _listOfNumberToCount = ['1', '2', '3'];
+        _illegalalCharacters = ['-', '<', '>'];
+    }
+
+    public bool IsCharacterAndIfINeedToCount(char character)
+    {
+        if (char.IsAsciiLetter(character)) 
+        {
+            var letter = character;
+
+            if (_listOfLettersToCount.Contains(letter))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        return false;
+    }
+
+    public bool IsDigitAndIfItIsInTheList(char character)
+    {
+        if (char.IsDigit(character))
+        {
+            var letter = character;
+
+            if (_listOfLettersToCount.Contains(letter))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        return false;
+    }
+
+    public void IncreaseDigitCount(int digitCount, bool b)
+    {
+        if (b)
+        {
+            digitCount++;
```

Comment:
what do you mean ? 
From the main method I pass digitCount variable for it to be increased based on boolean

Reply:
Value types, such as int, are passed by value, not reference, so a local copy of digitCount is created with the value passed during the call, then incremented, and then discarded, the value passed to the method call remains unchanged. You could pass it as a reference using either the ['ref' or 'out' keywords](https://www.c-sharpcorner.com/UploadFile/ff2f08/ref-vs-out-keywords-in-C-Sharp/). However, I would recommend simply returning int from the method, since it will be the most efficient if we're resource-constrained (4 bytes of passing int value, vs 8 bytes of passing a reference every time).